plop-templates/basic-component/src/ComponentName/ComponentName.spec.tsx.hbs

```diff
@@ -0,0 +1,19 @@
+import React from "react"
+import { render } from "@testing-library/react"
+import { {{pascalCase componentName}}, {{pascalCase componentName}}Props } from "./{{pascalCase componentName}}"
+
+const DEFAULT_PROPS: {{pascalCase componentName}}Props = {
+  /** @todo: Add default props for your test component */
+}
+
+const render{{pascalCase componentName}} = (customProps?: Partial<{{pascalCase componentName}}Props>) => {
+  const props = { ...DEFAULT_PROPS, ...customProps }
+  return render(<{{pascalCase componentName}} {...props} />)
+}
+
+describe("<{{pascalCase componentName}} />", () => {
+  it("test case description goes here", () => {
+    const { container } = render{{pascalCase componentName}}()
+    /** @todo: Fill in test case */
```

Comment:
What about adding a `expect(true).toBe(false)` here? That way you cannot forget about this file ;)